app_dart/test/request_handlers/refresh_github_commits_test.dart

```diff
@@ -47,9 +47,13 @@ void main() {
         final User author = User()
           ..login = 'Username'
           ..avatarUrl = 'http://example.org/avatar.jpg';
+        final GitCommitUser committer = GitCommitUser('Username','Username@abc.com',DateTime.now());
```

Comment:
Maybe we can use some earlier DateTime here, and have the test later assert that it sends the right date to the datastore?

Reply:
Added.